webapp/lib/app/configurator/tags/view.dart

```diff
@@ -6,6 +6,7 @@ import 'package:dnd/dnd.dart' as dnd;
 import 'package:katikati_ui_lib/components/accordion/accordion.dart';
 import 'package:katikati_ui_lib/components/editable/editable_text.dart';
 import 'package:katikati_ui_lib/components/button/button.dart';
+import 'package:katikati_ui_lib/components/model/model.g.dart';
```

Comment:
Nit: don't import the generated model directly, but instead import the wrapper `model.dart`
```suggestion
import 'package:katikati_ui_lib/components/model/model.dart';
```